tests/Aspire.TestUtilities/ActivityNotifier.cs

```diff
@@ -6,7 +6,7 @@
 using System.Threading.Channels;
 using OpenTelemetry;
 
-namespace Aspire.Components.Common.Tests;
```

Comment:
Does ActivityNotifier belong in the original project instead?

Reply:
I think you're right, it probably belongs more to the original project. Moved back.